src/mixins/datetime.js

```diff
@@ -92,7 +92,10 @@ export default {
          */
         datetimeFormat(value, format) {
             if (value !== undefined && value !== "") {
-                return dayjs.utc(value).tz(this.timezone).format(format);
+                const date = new Date(value);
+                return date.toLocaleString(this.$i18n.locale, { ...format,
+                    hour12: false,
+                    timeZone: this.timezone }); // 24 hours format
```

Comment:
This comment does not add additonal context

```suggestion
                    timeZone: this.timezone });
```